docs/lifecycle-events.md

```diff
@@ -25,7 +25,9 @@ Fired before the element is removed from the DOM.
 Note that when using this event you are responsible for removing the element yourself.
 
 ```js
-element.parent.removeChild(element)
+if (element.parentNode) {
```

Comment:
I think `if (element.parentNode) {` is not needed because the element is guaranteed to have a parent (the root property, which defaults to document.body if none is given).